config.toml

```diff
@@ -112,12 +112,27 @@ archived_version = false
 # The version number for the version of the docs represented in this doc set.
 # Used in the "version-banner" partial to display a version number for the
 # current doc set.
-version = "0.0"
+version = "v0.1.1"
 
 # A link to latest version of the docs. Used in the "version-banner" partial to
 # point people to the main doc site.
 url_latest_version = "https://cdevents.dev/docs"
 
+# Maintain history of release versions here
+# This section controls the content of the Releases menu
+# Note - modification to layouts/partials/breadcrumb.html to inject {{ .Site.Params.version }} / &nbsp; at root of breadcrumb path
+[[params.versions]]
+  version = "latest"
+  url = "https://cdevents.dev/docs"
+
+#[[params.versions]]
+#  version = "v0.1.1"
+#  url = "https://github.com/cdevents/spec/tree/v0.1.1"
+
+[[params.versions]]
+  version = "v0.1.0"
+  url = "https://github.com/cdevents/spec/tree/v0.1.0"
```

Comment:
This could point to https://github.com/cdevents/spec/tree/v0.1.0/spec.md

Reply:
I’m happy with that change, if you are.

Reply:
Implemented.